package.json

```diff
@@ -156,14 +157,30 @@
     "node-uuid": "1.4.7",
     "pegjs": "0.9.0",
     "postcss-loader": "1.2.1",
+    "pui-react-overlay-trigger": "^7.0.0",
+    "pui-react-tooltip": "^7.0.0",
     "querystring-browser": "1.0.4",
     "raw-loader": "0.5.1",
+    "react": "15.2.0",
+    "react-ace": "3.7.0",
+    "react-addons-test-utils": "15.2.0",
+    "react-anything-sortable": "^1.6.1",
+    "react-color": "^2.2.7",
+    "react-dom": "15.2.0",
+    "react-markdown": "^2.4.2",
+    "react-redux": "4.4.5",
+    "react-router": "2.0.0",
+    "react-router-redux": "4.0.4",
+    "react-select": "^1.0.0-rc.1",
+    "react-sortable": "^1.1.0",
+    "reactcss": "^1.0.7",
     "request": "2.61.0",
     "rimraf": "2.4.3",
     "rison-node": "1.0.0",
     "rjs-repack-loader": "1.0.6",
     "script-loader": "0.6.1",
     "semver": "5.1.0",
+    "simianhacker-react-resize-aware": "^1.0.11",
```

Comment:
We don't need to do it in this PR, but we should just write this ourselves tracking behavior ourselves.  The module is like 50 lines total and is clearly not actively maintained/used.